native/src/main/java/io/ballerina/stdlib/http/api/InterceptorResource.java

```diff
@@ -0,0 +1,200 @@
+package io.ballerina.stdlib.http.api;
```

Comment:
License header.

Reply:
Fixed in [`7f4cb18`](https://github.com/ballerina-platform/module-ballerina-http/pull/713/commits/7f4cb1838084bfbb629abafe3b04caa96ba1e09c)